build.gradle

```diff
@@ -1,37 +1,33 @@
 import org.jetbrains.dokka.gradle.DokkaTask
 
 buildscript {
-  ext {
-    kotlin_version = '1.8.22'
-  }
   repositories {
     google()
     mavenCentral()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:8.0.0'
-    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    classpath libs.android.gradle.plugin
   }
 }
 
 plugins {
-  id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
-  id("org.jetbrains.kotlin.android") version "$kotlin_version" apply false
-  id("org.jetbrains.dokka") version "1.9.20"
-  id("org.jlleitschuh.gradle.ktlint") version "11.5.0"
-  id("app.cash.paparazzi") version '1.3.1' apply false
+  alias libs.plugins.nexus.publish
+  alias libs.plugins.dokka
+  alias libs.plugins.kt.lint
+  alias libs.plugins.kotlin.android apply false
+  alias libs.plugins.paparazzi apply false
 }
 
 allprojects {
   repositories {
     google()
     mavenCentral()
-    flatDir {
-      dirs '../libs'
-    }
 
     // Needed for some automated and manual testing (e.g: acceptance tests)
     mavenLocal()
+    maven {
+      url "https://s01.oss.sonatype.org/content/repositories/comglia-1291/"
```

Comment:
Does it make sense to add a TODO here so that we don't forget to remove this before release?